ota-plus-web/app/com/advancedtelematic/auth/garage/LoginAction.scala

```diff
@@ -22,6 +25,39 @@ class LoginAction @Inject()(conf: Configuration,
   }
 }
 
+class LogoutAction @Inject()(conf: Configuration,
+                             ws: WSClient,
+                             authAction: UiAuthAction,
+                             val parser: BodyParsers.Default,
+                             val executionContext: ExecutionContext)
```

Comment:
you could make this implicit with `)(implicit val executionContext: ExecutionContext)` so you don't have to specify it below